packages/bot-web-ui/src/components/notify-item/__tests__/notify-item.spec.tsx

```diff
@@ -36,4 +36,59 @@ describe('messageWithButtonMockProps', () => {
         expect(screen.getByText('sample text')).toBeInTheDocument();
         expect(screen.getByRole('img')).toBeInTheDocument();
     });
+
+    it('777should render messageWithImage', () => {
```

Comment:
@maryia-matskevich-deriv I think this one is a typo

Reply:
sure, thanks for your attentiveness🙏

Reply:
✅corrected

Reply:
@maryia-matskevich-deriv kindly help to resolve the comments after fixing